apps/fabric-website/homepage.htm

```diff
@@ -17,52 +19,63 @@
 
 <div id="main">
   <div class="loading">
-    <img class="loadingImage" src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/loading.gif" alt="Loading"
-      width="32" height="32" />
+    <img class="loadingImage" src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/spinner.gif"
+      alt="Loading" width="32" height="32" />
   </div>
 </div>
 
 <script type="text/javascript">
-  var isProduction = false;
-  var isDogfood = false;
-  var now = Date.now();
-  var today = new Date();
-  var todaysDate = today.getFullYear() + '-' + (today.getMonth() + 1) + '-' + today.getDate();
+  var noCompress = getParameterByName('noCompress');
+  var devhost = getParameterByName('devhost');
   var entryPointFilename = 'fabric-sitev5';
```

Comment:
Dead variable? (at least, it isn't referenced elsewhere below)

Reply:
Ah yup--part of in-progress work but not implemented yet. Can remove for now.